### ISSUE TITLE
license question

### ISSUE THREAD

**Issue by IzzySoft** [wanam/YouTubeAdAway#116]
Could you please specify what license your app falls under? None being specified means "all rights reserved" and thus lacks some clarity. Thanks in advance!

**Comment by wanam** [comment]
Done! Thanks for the heads-up.

**Comment by IzzySoft** [comment]
Thanks for the fast reaction – and for the "kind of reaction" :heart_eyes: Let me give you a second heads-up then: Thanks to the license now there, I just included your app with my repo (with xposed.info being down I descided to give Xposed modules more room there). It will [show up here](https://apt.izzysoft.de/fdroid/index/apk/ma.wanam.youtubeadaway) in about an hour (when the next sync is through). Be welcome to [pick a badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) to link there from your Readme if you wish (no requirement of course, just an offer :wink:)

PS: in case you wonder, the "NonFreeAdd" is as the module is a helper to a non-libre app, Youtube. F-Droid users know how to read that, so no worries: if they are using Youtube, they'd expect that flag being there – and if not… well, they won't be interested anyway, right? :rofl:

So: A big thanks again!
